uiowa/uids
# Feature: Events/footer/safari fixes

Resolves #89, resolves #90, resolves #91, resolves #93, resolves #95, resolves #63

# How to test #90

View changes at http://localhost:3000/components/preview/events-block
- Resize page to view responsive adjustments (note: some margin spacing for these blocks can only be viewed when used within layout builder)
- Run through checkboxes on #90 issue page and verify that they have been made
- Verify that event title link has hover and focus behavior

# How to test #89 

Go to http://localhost:3000/components/detail/uiowa
- Verify that Search text doesn't hop up and down
- Verify that Search and menu and top links are aligned as close as possible

# How to test #91 

Go to http://localhost:3000/components/detail/uiowa
- Verify that socket links are split into two lines
- Verify that copyright year is being outputted by Twig function
- Verify that "Accessibility" link has been added
- Verify that all links in socket menu are correct and `https:`
- Verify that font awesome psuedo phone icon is gone from phone number and that icons are being added through `<i>` tags. 

# How to test #93 

Open http://localhost:3000/components/detail/uiowa
- Verify when resizing browser that Block Iowa logo remains centered within tab

# How to test #95

Open http://localhost:3000/components/detail/uiowa
- Verify that "Welcome Home" Section appears on black background with headline, copy, and button centered and all on separate lines.
